homeassistant/components/mysensors.py

```diff
@@ -125,7 +124,8 @@ def mysensors_callback(gateway, node_id):
         for child in gateway.sensors[node_id].children.values():
             for value_type in child.values.keys():
                 key = node_id, child.id, value_type
-                if child.type not in s_types or value_type not in v_types:
+                if child.type not in map_sv_types or \
```

Comment:
You're using a `defaultdict` so there is no need to test if `child.type` in `map_sv_types`